src/offline/cable_mpimaster.F90

```diff
@@ -332,922 +337,933 @@ SUBROUTINE mpidrv_master (comm, trunk_sumbal, dels, koffset, kend, PLUME, CRU)
     ! outer loop - spinup loop no. ktau_tot :
     ktau     = 0
     SPINLOOP:DO
-       YEARLOOP: DO YYYY= CABLE_USER%YearStart,  CABLE_USER%YearEnd
+      YEARLOOP: DO YYYY= CABLE_USER%YearStart,  CABLE_USER%YearEnd
+
+        CurYear = YYYY
+
+        !ccc Set calendar attribute: dependant on the value of `leaps`
+        ! that is set in the MetType if conditions above.
+        calendar = "noleap"
```

Comment:
Since the IF statements have moved to the cable_driver_common_mod.F90, updating the comment:


```suggestion
        !ccc Set calendar attribute: dependant on the value of `leaps`
        ! dependant on the MetType and set during initialisation.
        calendar = "noleap"
```